taxi/views.py

```diff
@@ -1,3 +1,21 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from django.template import loader
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request):
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
+
+    template = loader.get_template("taxi/index.html")
+
+    return HttpResponse(template.render(context, request))
```

Comment:
I don't think it is a mistake. but you could just use render(request, "template-path", context)

Reply:
@wQuelS btw, the task said that you have to return `HttpResponse`

Reply:
but two ways are good

Reply:
> @wQuelS btw, the task said that you have to return `HttpResponse`

render() combines a given template with a given context and returns an HttpResponse.
So, technically, using render() is doing according to the given task :)